editor/editor_zoom_widget.cpp

```diff
@@ -178,7 +178,7 @@ EditorZoomWidget::EditorZoomWidget() {
 	zoom_reset->add_theme_color_override("font_outline_color", Color(0, 0, 0));
 	zoom_reset->add_theme_color_override("font_color", Color(1, 1, 1));
 	zoom_reset->connect("pressed", callable_mp(this, &EditorZoomWidget::_button_zoom_reset));
-	zoom_reset->set_shortcut(ED_SHORTCUT("canvas_item_editor/zoom_reset", TTR("Zoom Reset"), KeyModifierMask::CMD | Key::KEY_0));
+	zoom_reset->set_shortcut(ED_GET_SHORTCUT("canvas_item_editor/zoom_reset"));
```

Comment:
Shouldn't it get `zoom_100_percent` then?

Reply:
Ah, right...